task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -148,6 +154,9 @@ spec:
         echo "$BUILD_DIR"
         ssh -v $SSH_ARGS "$SSH_HOST"  mkdir -p "$BUILD_DIR/workspaces" "$BUILD_DIR/scripts" "$BUILD_DIR/tmp" "$BUILD_DIR/tekton-results" "$BUILD_DIR/entitlement"
 
+        # ensure that a config toml file is present in case one is not provided
+        touch /var/workdir/source/$CONFIG_TOML_FILE
```

Comment:
would that work if someone specify `CONFIG_TOML_FILE=a/b/c/config.toml` but none of that exist ?

It should probably fail anyway

Reply:
It will not work, no. Not without the `mkdir -p`.